github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
+  kubevirt/common-instancetypes:
+    - name: publish-common-instancetypes-builder
+      
```

Comment:
Publish jobs looks good - passed here: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-common-instancetypes-builder/1588531214032048128

Just no need for this empty line.

Reply:
ACK removed, thanks.